### PR TITLE
New pages: HTMLInputElement checkedness

### PR DESCRIPTION
**Pull request by estelle**
part of https://github.com/mdn/content/issues/35544

### CODE REVIEW

File: files/en-us/web/api/htmlinputelement/checked/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title: "HTMLInputElement: checked property"
+short-title: checked
+slug: Web/API/HTMLInputElement/checked
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.checked
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`checked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the current checkedness of the element; whether the form control is checked or not. It reflects the {{htmlelement("input")}} element's [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/checked/index.md:11]:
```suggestion
The **`checked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the current checkedness of the element; that is, whether the form control is checked or not. It reflects the {{htmlelement("input")}} element's [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute.
```

File: files/en-us/web/api/htmlinputelement/checked/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The boolean `checked` attribute is valid for the `radio`([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/checked/index.md:13]:
```suggestion
The boolean `checked` attribute is valid for the `radio` ([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
```

File: files/en-us/web/api/htmlinputelement/checked/index.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+A boolean attribute indicating whether this checkbox is checked by default (when the page loads). It does not indicate whether this checkbox is currently checked: if the checkbox's state is changed, this content attribute does not reflect the change. (Only the HTMLInputElement's checked IDL attribute is updated.)
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/checked/index.md:15]: Then it does not "reflect" the attribute. An attribute is only "reflected" by a property if getting/setting the property is equivalent to getting/setting the attribute. Indeed, the `checked` attribute is reflected by the `defaultChecked` property, just like the `value` attribute is reflected by the `defaultValue` property.

File: files/en-us/web/api/htmlinputelement/defaultchecked/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+title: "HTMLInputElement: defaultChecked property"
+short-title: defaultChecked
+slug: Web/API/HTMLInputElement/defaultChecked
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.defaultChecked
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`defaultChecked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the default checkedness state of the element. This property represents the {{htmlelement("input")}} element's default[`checked`](/en-US/docs/Web/HTML/Element/input#checked) state as originally specified in HTML that created this object.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/defaultchecked/index.md:11]:
```suggestion
The **`defaultChecked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the default checkedness state of the element. This property reflects the {{htmlelement("input")}} element's [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute.
```

(Perhaps you could further clarify that the `checked` attribute in fact only sets the default state.)

File: files/en-us/web/api/htmlinputelement/defaultchecked/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+The boolean `checked` attribute is valid for the `radio`([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/defaultchecked/index.md:13]:
```suggestion
The boolean `checked` attribute is valid for the `radio` ([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/htmlinputelement/checked/index.md (hunk 3 above)
Comment (estelle) [reply files/en-us/web/api/htmlinputelement/checked/index.md]: updated!

File: files/en-us/web/api/htmlinputelement/defaultchecked/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+slug: Web/API/HTMLInputElement/defaultChecked
+page-type: web-api-instance-property
+browser-compat: api.HTMLInputElement.defaultChecked
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`defaultChecked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the default checkedness state of the element. This property reflects the {{htmlelement("input")}} element's [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute.
+
+The boolean `checked` attribute is valid for the `radio` ([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
```
Comment (estelle) [inline files/en-us/web/api/htmlinputelement/defaultchecked/index.md:13]:
```suggestion
The boolean [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute is valid for the `radio` ([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types. The presence of the attribute sets the `defaultChecked` property to `true`.
```

File: files/en-us/web/api/htmlinputelement/checked/index.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+browser-compat: api.HTMLInputElement.checked
+---
+
+{{ APIRef("HTML DOM") }}
+
+The **`checked`** property of the {{DOMxRef("HTMLInputElement")}} interface specifies the current checkedness of the element; that is, whether the form control is checked or not.
+
+The boolean `checked` property is relevant to the `radio` ([`<input type="radio">`](/en-US/docs/Web/HTML/Element/input/radio)) and `checkbox` ([`<input type="checkbox">`](/en-US/docs/Web/HTML/Element/input/checkbox)) input types.
+
+The presence of the boolean [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute indicates the checkbox is checked by default. It does not indicate whether this checkbox is currently checked: if the checkbox's state is changed, this content attribute does not reflect the change; only the `HTMLInputElement`'s `checked` IDL attribute is updated.
```
Comment (Josh-Cena) [inline files/en-us/web/api/htmlinputelement/checked/index.md:15]:
```suggestion
The presence of the HTML [`checked`](/en-US/docs/Web/HTML/Element/input#checked) attribute indicates the checkbox is checked by default. It does not indicate whether this checkbox is currently checked: if the checkbox's state is changed, this content attribute does not reflect the change; only the `HTMLInputElement`'s `checked` IDL property is updated. The `checked` attribute is reflected by the {{domxref("HTMLInputElement.defaultChecked", "defaultChecked")}} property.
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**